frontend/src/app/corpus-definitions/definitions-overview/definitions-overview.component.ts

```diff
@@ -15,13 +18,33 @@ export class DefinitionsOverviewComponent {
 
     corpora$: Observable<APIEditableCorpus[]>;
 
-    constructor(private apiService: ApiService) {
+    corpusToDelete$ = new BehaviorSubject<APIEditableCorpus|null>(null);
+    deleteLoading$ = new BehaviorSubject<boolean>(false);
+
+    constructor(
+        private apiService: ApiService,
+        private title: Title
+    ) {
         this.corpora$ = this.apiService.corpusDefinitions();
+        this.title.setTitle(pageTitle('Corpus definitions'));
```

Comment:
I take this is for accesbility reasons? If we repeat this behaviour a lot, we could subclass component to take a argument (similar to `selector`) that contains the title.